causync.py

```diff
@@ -149,17 +179,25 @@ def get_basename(self, path):
 
         return basename
 
-    def get_lockfile_path(self):
+    def check_lockfiles(self):
+        """ check if logfiles exist for all source directories """
```

Comment:
For documentation strings, please use full sentences, start with a capital letter (C) and finish with a dot (.).